src/components/signup/SignUpForm.js

```diff
@@ -16,30 +23,77 @@ const SignUpForm = () => {
       text: intl.formatMessage({ id: "form.other" }),
     },
   ];
+
+  const [user, setUser] = useState({
+    username: "",
+    email: "",
+    password: "",
+    password_confirmation: "",
```

Comment:
Variables in Javascript are written in camelCase 
```suggestion
    passwordConfirmation: "",
```